web/src/alerts.ts

```diff
@@ -7,6 +7,7 @@ export type Alert = {
   msg: string
   timestamp: string
   resourceName: string
+  dismissHandler?: () => void
```

Comment:
When is this not set on the alert?

Reply:
most alerts don't have a way to dismiss them